### PR TITLE
[SWPWA-356] Fix email autocomplete in checkout

### PR DESCRIPTION
**Pull request by negzu**

### CODE REVIEW

File: src/app/component/Field/Field.component.js [hunk 1]
```diff
@@ -391,6 +391,7 @@ export default class Field extends PureComponent {
               name={ name }
               value={ value }
               disabled={ isDisabled }
+              form={ form }
```
Comment (alfredsgenkins) [inline src/app/component/Field/Field.component.js]: I think this should be added to all input types, not just one you added it to.

File: src/app/component/Input/Input.style.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.Input {
```
Comment (Danpiel) [inline src/app/component/Input/Input.style.scss]: Codacy Issue found: [Unexpected empty block (block-no-empty)](https://app.codacy.com/manual/ScandiPWA/base-theme/pullRequest?prid=4724211)

**Comment by Danpiel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           

Complexity increasing per file
==============================
- src/app/component/Input/Input.container.js  4
- src/app/component/Input/Input.component.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ScandiPWA/base-theme/pullRequest?prid=4724211&bid=15772692)

File: src/app/component/Input/Input.style.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+
```
Comment (Danpiel) [inline src/app/component/Input/Input.style.scss]: Codacy Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/manual/ScandiPWA/base-theme/pullRequest?prid=4724211)

File: src/app/component/Input/Input.style.scss [hunk 4]
```diff
@@ -0,0 +1,3 @@
+.Input {
+    
+}
```
Comment (Danpiel) [inline src/app/component/Input/Input.style.scss]: Codacy Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/manual/ScandiPWA/base-theme/pullRequest?prid=4724211)

**Comment by negzu** [comment]
After code review changes were applied. See more in commit descriptions and code itself :)

**Comment by Danpiel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/app/component/Input/Input.component.js  1
- src/app/component/Input/Input.container.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ScandiPWA/base-theme/pullRequest?prid=4724211&bid=15772692)

**Pull request merged**